waspc/data/Generator/templates/sdk/wasp/server/auth/hooks.ts

```diff
@@ -0,0 +1,46 @@
+import type { Request as ExpressRequest } from 'express'
+import type { ProviderId, createUser } from '../../auth/utils.js'
+import { prisma } from '../index.js'
+
+type CommonInput = {
```

Comment:
Each hook will receive `hookName`, `prisma` and Express's `req` object.

Reply:
Renamed the symbol and added the `Expand` helper so user would see full types
![Screenshot 2024-06-05 at 16 51 33](https://github.com/wasp-lang/wasp/assets/2223680/dce480d8-5998-432c-88be-0d9bff2cecc3)